### PR TITLE
Tensorflow works. 

### PR DESCRIPTION
**Pull request by jcchurch**
There's a problem with face007 in training being recognized as "not face".

- Removed the dependence on global variables.
- Images are now 3 digits in the filename instead of the original 4. We may have to go back to 4.)
- Divided the image set into training images and test images. (12 in training set, 6 in the test set)
- 5 of the 6 test images were properly recognized. 83% accuracy!

### CODE REVIEW

**Comment by MichaelTimbes** [comment]
Found the instructions to merge the pull request.
